<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>323</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>&amp;基本参数：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>分布维度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dims2">
          <property name="text">
           <string>二维</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dims3">
          <property name="text">
           <string>三维</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>迭代次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="iterTimes_S">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="iterTimes">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonMore">
     <property name="text">
      <string>&amp;More</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>&amp;详细参数：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>引力系数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attractParam">
          <property name="text">
           <string>100.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>斥力系数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="repulsParam">
          <property name="text">
           <string>0.08</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>降温系数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="temperatureK">
          <property name="text">
           <string>0.8</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>单位位移：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dispMove">
          <property name="text">
           <string>0.001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>布局细节：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="oneNode">
          <property name="text">
           <string>逐节点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="oneProcess">
          <property name="text">
           <string>逐过程</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMore</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterTimes_S</sender>
   <signal>valueChanged(int)</signal>
   <receiver>iterTimes</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterTimes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>iterTimes_S</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMore</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>showExtension(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
